lib/form-utils.js

```diff
@@ -49,7 +50,7 @@ export const requireFields = (data, requiredFields, { stopOnFirstError = false }
 export const verifyFieldLength = (intl, errors, data, field, min, max) => {
   // Ignore if there's already an error on the field
   if (!errors[field]) {
-    const length = data[field]?.length || 0;
+    const length = get(data, field).length || 0;
```

Comment:
Though it's great that you added support for subfields with `get`, this will crash if the value for the field is `null` or `undefined`. To make sure we don't introduce any bug, we should do:

```suggestion
    const length = get(data, field)?.length || 0;
```

Reply:
Alternative: use lodash's `size`:

```es6
const length = size(get(data, field));
```